crate/cli/src/actions/cover_crypt/keys/revoke_key.rs

```diff
@@ -14,19 +14,35 @@ use crate::{actions::shared::utils::revoke, error::CliError};
 /// Once a user decryption key is revoked, it will no longer be rekeyed
 /// when attributes are rotated on the master key.
 ///
+/// When using tags to revoke the key, rather than the key id,
+/// an error is returned if multiple keys matching the tags are found.
```

Comment:
Because the KMIP revoke, operation is explicitly specified as revoking a single key.
If we change that behavior, we must change the whole documentation around it and mention that we do not follow the specifications. Also, this may be dangerous.

BTW, this is an issue for Encrypt and Decrypt, where support for bulk is needed.